com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -432,6 +432,12 @@ int GetDestination()
                     m_Materials.uber.EnableKeyword(ShaderKeywordStrings.UseFastSRGBLinearConversion);
                 }
 
+                if(isSceneViewCamera)
+                {
+                    // Scene-view doesn't necessarily use the final-blit, so ensure we set the debug properties here (if necessary)...
+                    DebugHandler?.UpdateShaderGlobalPropertiesFinalBlitPass(cmd, ref cameraData);
+                }
```

Comment:
Game view also doesn't necessarily use FinalBlit pass. 
 Also, shouldn't this be done only for the post FinalBlitPass?

Reply:
I think the idea is to call this function for whatever pass is the last one. But you're right this logic might not be correct as the conditions under which FinalBlitPass is used are more complicated than this. I'm having a hard time figuring out what's the right thing here, @andrewt-unity could you explain this logic a bit?